packages/types/src/debugMeta.ts

```diff
@@ -3,6 +3,7 @@
  **/
 export interface DebugMeta {
   images?: Array<DebugImage>;
+  transactionSampling?: { rate?: number; method?: string };
```

Comment:
If we call it `rate` and `id` in the protocol, couldn't we unify it here as well? It'd make some object deconstruction easier.

Reply:
It would make object deconstruction easier. I didn't do it because `id` is opaque. What's it the id of? Unless you feel strongly I'd rather keep it as is.

Reply:
I totally agree, but why is Relay using it in the first place? :P

Reply:
You'll have to take that up with @jan-auer 😛

Reply:
See also https://github.com/getsentry/sentry-javascript/pull/3197#discussion_r566627127. Relay doesn't use this field, but it also sample in its own way. The `id` field identifies not only the method but in case of Relay also the rule which applied. 

Ultimately, this all ends up in Sentry and in our data pipeline.